pkg/sql/repair.go

```diff
@@ -569,7 +567,7 @@ func (p *planner) UnsafeDeleteDescriptor(ctx context.Context, descID int64, forc
 	mut, err := p.Descriptors().GetMutableDescriptorByID(ctx, id, p.txn)
 	var forceNoticeString string // for the event
 	if err != nil {
-		if !errors.Is(err, catalog.ErrDescriptorNotFound) && force {
+		if force {
```

Comment:
I think `force` is enough here. The other test was failing because ErrDescriptorNotFound was being returned by a failing back-reference check.